Sources/TSCBasic/PathShims.swift

```diff
@@ -23,12 +23,12 @@ import Foundation
 /// Returns the "real path" corresponding to `path` by resolving any symbolic links.
 public func resolveSymlinks(_ path: AbsolutePath) -> AbsolutePath {
 #if os(Windows)
-    let resolved: String =
-        (try? FileManager.default.destinationOfSymbolicLink(atPath: path.pathString))
-            ?? path.pathString
+    var resolved: URL = URL(fileURLWithPath: path.pathString)
+    if let destination = try? FileManager.default.destinationOfSymbolicLink(atPath: path.pathString) {
+        resolved = URL(fileURLWithPath: destination, relativeTo: URL(fileURLWithPath: path.pathString))
```

Comment:
In theory, could we use `AbsolutePath.init(_ str: String, relativeTo basePath: AbsolutePath)` here, which takes a string such as what `destinationOfSymbolicLink` returns (either relative or absolute path) and returns the resulting absolute path?

Reply:
We could, but we want the standardized URL ... so we would need to construct the URL first anyway.